components/x-teaser/src/Image.jsx

```diff
@@ -25,26 +25,30 @@ const LazyImage = ({ src, lazyLoad }) => {
 }
 
 export default ({ relativeUrl, url, image, imageSize, imageLazyLoad, imageHighestQuality, ...props }) => {
-	const displayUrl = relativeUrl || url
-	const useImageService = !(image.url.startsWith('data:') || image.url.startsWith('blob:'))
-	const options = imageSize === 'XXL' && imageHighestQuality ? { quality: 'highest' } : {}
-	const imageSrc = useImageService ? imageService(image.url, ImageSizes[imageSize], options) : image.url
-	const ImageComponent = imageLazyLoad ? LazyImage : NormalImage
+	if (image) {
+		const displayUrl = relativeUrl || url
+		const useImageService = !(image.url.startsWith('data:') || image.url.startsWith('blob:'))
+		const options = imageSize === 'XXL' && imageHighestQuality ? { quality: 'highest' } : {}
+		const imageSrc = useImageService ? imageService(image.url, ImageSizes[imageSize], options) : image.url
+		const ImageComponent = imageLazyLoad ? LazyImage : NormalImage
 
-	return image ? (
```

Comment:
Question: Wasn't the ternary operator applied here producing the same effect? I mean that if `image` is undefined it should return null.

Reply:
line 29 causes issues. we need to check `image` before the line
https://sentry.io/organizations/financial-times/issues/2005812713/?project=79503

Reply:
Yep @fenglish has got it - this code runs before we check for the presence of `image` and errors with `Cannot read property 'startsWith' of undefined`:

```js
const useImageService = !(image.url.startsWith('data:') || image.url.startsWith('blob:'))
const options = imageSize === 'XXL' && imageHighestQuality ? { quality: 'highest' } : {}
const imageSrc = useImageService ? imageService(image.url, ImageSizes[imageSize], options) : image.url
```

Reply:
`Cannot read property 'startsWith' of undefined` means that the `url` is undefined, not `image`. Image could be defined but not having a `url`. If Image is `undefined` how is that bit of code executed? The ternary operator should prevent that. Is it possible that `image` exists but doesn't have `url`?

Reply:
If you run the existing function with no argument I would expect to return null, if you run it with an empty literal object like `{}` I would expect you to see the same error, cause `url` will not exists

Reply:
Sorry, I was wrong about the ternary cause that is executed after, I misread in the rush. Pardon me 🙏